README.rst

```diff
@@ -49,6 +49,88 @@ You can run all of the tox environments using the command ``tox``.
 
 Flocker is also tested using `continuous integration`_.
 
+Cloudformation Deployment
+-------------------------
+
+* Generate a Cloudformation template:
+
+```
+$ python ./admin/installer/cloudformation.py > flocker-cloudformation-template.json
```

Comment:
:grey_question: with these instructions, I think we should either put `flocker-cloudformation-template.json` into our .gitignore, or instead place `flocker-cloudformation-template.json` off in `/tmp/`